### ISSUE TITLE
pa11y-ci GitHub Actions execution error: badge component

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/Orange-Boosted-Bootstrap#760]
Due to the previous events related to pa11y-ci execution, one issue was masked coming from 5.0.2.

If I modify `build/.pa11yci.json` to set the concurrency to 1, I can see the following error into `pa11y-ci-results.json`:

```json
  "http://localhost:9001/docs/5.0/components/badge/": [
      {
        "code": "color-contrast",
        "type": "error",
        "typeCode": 1,
        "message": "Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=axeAPI)",
        "context": "<span class=\"position-absolute top-0 start-100 translate-middle badge rounded-pill bg-info text-white\">\n    99+\n    <span class=\"visua...</span>",
        "selector": "html > body > div:nth-child(6) > main > div:nth-child(3) > div:nth-child(16) > a > span:nth-child(3)",
        "runner": "axe",
        "runnerExtras": {
          "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
          "impact": "serious",
          "help": "Elements must have sufficient color contrast",
          "helpUrl": "https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=axeAPI"
        }
      }
    ],
```

The issue is visible here: https://boosted.orange.com/docs/5.0/components/badge/#positioned

This error should be hidden or fixed.

**Comment by julien-deramond** [comment]
- [ ] Fix the problem so that all pa11y-ci GitHub Actions won't fail
- [ ] Talk about this contrast error to GDL
  - If they choose to keep it as it is, close this issue
  - If they choose to make a modification, develop it and remove the fix in `build/.pa11yci.json` to hide the problem
